### PR TITLE
tests: subsys: bluetooth: fast_pair: Add nRF54L15 PDK target 

### PR DESCRIPTION
**Pull request by mkapala-nordic**
Added nRF54L15 PDK target to Fast Pair tests.

Jira: NCSDK-26582

### CODE REVIEW

**Review by MarekPieta — APPROVED** [review]

**Review by alstrzebonski — APPROVED** [review]
Mbedtls crypto support will be added later. Am I correct?

**Comment by mkapala-nordic** [comment]
> Mbedtls crypto support will be added later. Am I correct?

Discussed offline, there won't be Fast Pair MbedTLS crypto support for nRF54 series.

File: tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml [hunk 1]
```diff
@@ -15,22 +15,26 @@ tests:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
       - nrf5340dk_nrf5340_cpuapp_ns
+      - nrf54l15pdk_nrf54l15_cpuapp
```
Comment (MarekPieta) [inline tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml:18]: `nrf54l15pdk_nrf54l15_cpuapp_ns` support was introduced recently. Can we add it for the tests?

**Review by MarekPieta — APPROVED** [review]

File: tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml (hunk 1 above)
Comment (mkapala-nordic) [reply tests/subsys/bluetooth/fast_pair/crypto/testcase.yaml:18]: Discussed offline, we will do it in the other ticket: https://nordicsemi.atlassian.net/browse/NCSDK-26892

**Review by alstrzebonski — APPROVED** [review]

**Pull request merged**